screenpipe-audio/src/vad_engine.rs

```diff
@@ -19,9 +19,9 @@ pub enum VadSensitivity {
 impl VadSensitivity {
     pub fn min_speech_ratio(&self) -> f32 {
         match self {
-            VadSensitivity::Low => 0.01,     // 1% of frames must be speech
+            VadSensitivity::Low => 0.01,    // 1% of frames must be speech
```

Comment:
@EzraEllette just meta: is there a way everyone contributing has the same. config for linting so it's easier to review PRs and don't have small changes like this?